api/src/controllers/suite/lookup.ts

```diff
@@ -59,7 +59,10 @@ async function suiteLookup(
   // prepare a draft of our response with fields that depend on existence of
   // batches, set to undefined or empty array.
 
-  const output: SuiteLookupResponse = {
+  const output: SuiteLookupResponse & {
+    isSubscribed: boolean
+    subscriberCount: number
+  } = {
```

Comment:
I think this shows that maybe `SuiteLookupResponse` in `schema.yaml` should include `isSubscribed` and `subscriberCount`. It must have been an oversight on my part that has caused this inconsistency.

Reply:
Yes, please.